types/block.go

```diff
@@ -186,17 +196,88 @@ type BlockV2 struct {
 // We need to wait for a few blocks to pass (`signature_rewards_max_delay`) to store the finality signers because we need a bit of time to get the block finality.
 type SingleBlockRewardedSignatures []uint16
 
+type BlockTransactionCategory uint
+
+const (
+	BlockTransactionCategoryMint BlockTransactionCategory = iota
+	BlockTransactionCategoryAuction
+	BlockTransactionCategoryInstallUpgrade
+	BlockTransactionCategoryLarge
+	BlockTransactionCategoryMedium
+	BlockTransactionCategorySmall
+)
+
+type BlockTransactionVersion uint
+
+const (
+	BlockTransactionVersionV1 BlockTransactionVersion = iota
```

Comment:
I suggest to set Deploy=0, Version1=1 to match the values in casper-node
casper-node/types/src/transaction/transaction_header.rs:17